style.css

```diff
@@ -296,3 +296,111 @@ a {
   position: relative;
   margin-bottom: 16px;
 }
+
+.section-bg {
+  padding-top: 68px;
+  padding-bottom: 68px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-color: #f9f9fa;
+  background-image: url(./assets/geo.png);
+  background-repeat: no-repeat;
+}
+
+.section-bg h1 {
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  margin-bottom: 20px;
+}
+
+.section-bg label {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 15px;
+  padding: 12px 16px;
+}
+
+.form-border {
+  border: 1px solid #dfdfe0;
+  border-radius: 8px;
+  margin-bottom: 8px;
+  background-color: white;
+}
+
```

Comment:
- [ ] Kindly make sure to layout the form contents by using `flex-box` as requested in the project requirements.